mono/utils/mono-threads-coop.h

```diff
@@ -153,4 +153,17 @@ G_EXTERN_C // due to THREAD_INFO_TYPE varying
 gpointer
 mono_threads_enter_gc_unsafe_region_unbalanced_with_info (THREAD_INFO_TYPE *info, MonoStackData *stackdata);
 
+MONO_LLVM_INTERNAL_EXTERN_C_BEGIN
+extern char mono_threads_is_runtime_startup_finished_hidden_dont_modify MONO_LLVM_INTERNAL_NO_EXTERN_C;
+MONO_LLVM_INTERNAL_EXTERN_C_END
+
+static inline gboolean
+mono_threads_is_runtime_startup_finished (void)
+{
```

Comment:
These could go to runtime.h/runtime.c and named mono_runtime_is_startup_finished ().

Reply:
I need them in utils/ so that the threading state machine could call it.